source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -257,6 +259,22 @@ def _filter_too_much_connections(journeys, instance, request):
                 mark_as_dead(j, "too_much_connections")
 
 
+def _filter_min_transfers(journeys, instance, request):
+    """
+    eliminates journeys with number of connections more then max_nb_transfers among journeys
```

Comment:
Isn't it redundant with `_max_additional_connections`?
Looks like this line is a leftover of comment.
Maybe the 2 functions should be merged in one?